vmdb/app/helpers/container_helper/textual_summary.rb

```diff
@@ -13,6 +13,22 @@ def textual_group_properties
   #
 
   def textual_image
-    {:label => "image", :value => @record.image}
+    {:label => "Image", :value => @record.image}
+  end
+
+  def textual_name
+    {:label => "Name", :value => @record.name}
+  end
+
+  def textual_state
+    {:label => "State", :value => @record.state}
+  end
+
+  def textual_restart_count
+    {:label => "Restart Count", :value => @record.restart_count}
+  end
+
+  def textual_container_id
+    {:label => "Container ID", :value => @record.container_id}
```

Comment:
if you want to show the container unique id fromthe kubernetes side, I think you want to show ems_ref and that would be in properties.  I forget what we use for a label on ems_refs...you can check one of the other summary screens, probably.